lib/qb_integration/order.rb

```diff
@@ -42,10 +52,91 @@ def cancel
       [200, text]
     end
 
+    def as_flowlink_hash(qbo_order)
```

Comment:
why for address we got self.as_flowlink... and here as_flowlinkg_hash :thinking: ... Shouldnt QBO order be a instance var ?

Reply:
I was thinking about that and from the other parts of the repo, it looked like the instance vars are only being used with the payload, memoized or when initializing 🤔. This whole method might need to be it's own class